src/system/Notice/Notice.tsx

```diff
@@ -56,40 +60,139 @@ export const Notice = React.forwardRef< HTMLDivElement, NoticeProps >(
 			sx = {},
 			title,
 			variant = 'warning',
+			collapsible = false,
+			defaultOpen = false,
 			...props
 		},
 		forwardRef
 	) => {
+		const [ isExpanded, setIsExpanded ] = useState( defaultOpen );
+		const handleExpand = ( openValue: boolean ) => setIsExpanded( openValue );
+		const contentId = useId();
+
+		const iconColor = `notice.icon.${ variant }`;
+		const textColor = `notice.text.${ variant }`;
+
+		const baseCardSx: ThemeUIStyleObject = {
+			boxShadow: 'none',
+			borderRadius: 2,
+			bg: inline || collapsible ? 'transparent' : `notice.background.${ variant }`,
+			color: textColor,
+			fontSize: 2,
+			p: {
+				color: textColor,
+				fontSize: 2,
+			},
+			a: {
+				color: `notice.link.${ variant }.default`,
+				'&:visited': {
+					color: `notice.link.${ variant }.visited`,
+				},
+				'&:active': {
+					color: `notice.link.${ variant }.active`,
+				},
+				'&:hover, &:focus': {
+					color: `notice.link.${ variant }.hover`,
+				},
+			},
+			ul: {
+				pl: 5,
+			},
+			...sx,
+		};
+
+		if ( collapsible ) {
+			const renderHeader = () => (
+				<Collapsible.Trigger asChild aria-expanded={ isExpanded } aria-controls={ contentId }>
+					<button
+						type="button"
+						sx={ {
+							border: 'none',
+							width: '100%',
+							display: 'flex',
+							alignItems: 'center',
+							justifyContent: 'space-between',
+							cursor: 'pointer',
+							bg: `notice.background.${ variant }`,
+							px: 3,
+							py: 2,
+						} }
+					>
+						<Flex sx={ { alignItems: 'center', gap: 2 } }>
+							<NoticeIcon color={ iconColor } variant={ variant } />
+							<Heading
+								as={ headingVariant }
+								id={ `${ contentId }-heading` }
+								sx={ {
+									color: textColor,
+									fontSize: 2,
+									fontWeight: 'bold',
+									my: 2,
+									mx: 3,
+								} }
+							>
+								{ title }
+							</Heading>
+						</Flex>
+						<BiChevronDown
```

Comment:
I think we should use `<BiChevronDown>` or `<BiChevronUp>` based on `isExpanded`, instead of rotation

Reply:
And also add `data-arrow-indicator` and `aria-hidden="true"` props